chain/client/src/sync.rs

```diff
@@ -464,6 +421,12 @@ impl StateSync {
         }
     }
 
+    // TODO MOO ask Alex for this
+    pub fn get_num_parts(&self, state_root_node: &StateRootNode) -> u64 {
+        let state_size = state_root_node.memory_usage;
+        state_size / 1000000 + 3
```

Comment:
The only think is I would divide by `1048576` instead? But otherwise DLing in 1MiB chunks sounds reasonable.

Reply:
Discussed offline. I'll add a comment here.